doc/develop/extending.rst

```diff
@@ -193,46 +193,51 @@ of wxPython, which must be 3.x, and OpenCV, which must be 2.4.
     the Odemis virtualenv.
```

Comment:
I think you need to update the paragraphs above. Update to Python 3, remove wxPython restriction to using v3.x, and OpenCV up to 3.4 is fine.

Reply:
I think we can delete the entire paragraph. I never used easy_install during the installation.